plasmapy/particles/ionization_state_collection.py

```diff
@@ -162,8 +166,11 @@ def __init__(
                     )
                 set_abundances = False
 
+        def return_none():
+            return None
```

Comment:
Doesn't seem like that one's actually used - see codecov warning!

Reply:
Oh, cool — that means we can use a regular `dict` instead of a `defaultdict` when storing the parameters for `IonizationStateCollection`.